msr_rr_cfp.md

```diff
@@ -0,0 +1,83 @@
+# Title (required)
+Provide the working title of your study. It may be the same title that you submit for publication of your final manuscript, but it is not a requirement.
+* **Example**:: Should your family travel with you on the enterprise?
+* Subtitle (optional): Effect of accompanying families on the work habits of crew members
+
+# Authors (required)
+At this stage, we believe that an unblinded/single blind review is most productive
+
+# Structured Abstract (required)
+## Background/Context
+What is your research about? Why are you doing this research, why is it interesting?
+
+**Example**: The enterprise is the flag ship of the federation, and it allows families to travel onboard. However, there are no studies that evaluate how this affects the crew members”
+
+## Objective/Aim
+What exactly are you evaluating? What are the objects of the study?
+
+**Example**: We are studying whether the frequency of sick days, the work effectiveness and efficiency differ between science officers who bring their family with them, compared to science officers who are serving without their family
+
+## Method
+How are you addressing your objective?
+**Example**: We conduct an observational study and use a between subject design. To analyze the data, we use a t test or Wilcoxon test, depending on the underlying distribution
+
+## Early Results
+Here come the pilot data, if you have any
+
+## Limitations
+
+# Introduction (optional)
```

Comment:
Why make this optional? Seems it is always needed to give context and motivation?